### ISSUE TITLE
[Bug]: okhttp3 Response bodies must be closed

### ISSUE THREAD

**Issue by s-hintz** [flipt-io/flipt-server-sdks#114]
### Bug Description

Because auf Flipt, we get this log message:

`A connection to …  was leaked. Did you forget to close a response body? To see where this was allocated, set the OkHttpClient logger level to FINE: Logger.getLogger(OkHttpClient.class.getName()).setLevel(Level.FINE);`

See Response.kt from okhttp3:

```
/**
   * Returns a non-null value if this response was passed to [Callback.onResponse] or returned
   * from [Call.execute]. Response bodies must be [closed][ResponseBody] and may
   * be consumed only once.
   *
   * This always returns null on responses returned from [cacheResponse], [networkResponse],
   * and [priorResponse].
   */
  @get:JvmName("body") val body: ResponseBody?,
```

### Version Info

flipt-java-1.0.1.jar

### Search

- [X] I searched for other open and closed issues before opening this

### Steps to Reproduce

Use `FliptClient.java`.

### Expected Behavior

The response.body should be closed in `Evaluation.java`.

**Comment by markphelps** [comment]
Thanks @s-hintz ! 

I actually fixed this in #110 , however I just forgot to create a new release of the Java library. Doing that now! Will close this once the new version has been released to Maven Central

**Comment by markphelps** [comment]
@s-hintz this should now be fixed in [1.0.2](https://central.sonatype.com/artifact/io.flipt/flipt-java)

Thanks again for reporting!

**Comment by s-hintz** [comment]
Thank you very much for fixing!
